test/data/case3_dc.m

```diff
@@ -0,0 +1,85 @@
+% Case to test adding data to matpower file
+% based on nesta_case3_lmbd from NESTA v0.6.0
+function mpc = case3_dc
+mpc.version = '2';
+mpc.baseMVA = 100.0;
+
+mpc.bus = [
+	1	 3	 110.0	 40.0	 0.0	 0.0	 1	    1.10000	   -0.00000	 240.0	 1	    1.10000	    0.90000;
+	2	 2	 110.0	 40.0	 0.0	 0.0	 1	    0.92617	    7.25883	 240.0	 1	    1.10000	    0.90000;
+	3	 2	 95.0	 50.0	 0.0	 0.0	 1	    0.90000	  -17.26710	 240.0	 2	    1.10000	    0.90000;
+];
+
+mpc.gen = [
+	1	 158.066931397567	 28.7880705116472	 1000.0	 -1000.0	 1.1	 100.0	 1	 2000.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0;
+	2	 160.006267696445	-4.62694670645669	 1000.0	 -1000.0	 0.92617	 100.0	 1	 1500.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0;
+	3	 0.0	 -4.84256468513695	 1000.0	 -1000.0	 0.9	 100.0	 1	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0	 0.0;
+];
+
+mpc.gencost = [
+	2	 0.0	 0.0	 3	   0.110000	   5.000000	   0.000000;
+	2	 0.0	 0.0	 3	   0.085000	   1.200000	   0.000000;
+	2	 0.0	 0.0	 3	   0.000000	   0.000000	   0.000000;
+];
+
+mpc.branch = [
+	1	 3	 0.065	 0.62	 0.45	 9000.0	 0.0	 0.0	 0.0	 0.0	 1	 -30.0	 30.0;
+	3	 2	 0.025	 0.75	 0.7	 50.0	 0.0	 0.0	 0.0	 0.0	 1	 -30.0	 30.0;
+	1	 2	 0.042	 0.9	 0.3	 9000.0	 0.0	 0.0	 0.0	 0.0	 1	 -30.0	 30.0;
+];
+
+mpc.dcline = [
+	1	 2	 1	 10	 10	 25.9092634604825	 -4.16425203581105	 1.1	 0.92617	 10	 900 -900 900 -900 900	 0	 0	 0	 0	 0	 0	 0	 0
+]
+
+% matpower data format extentions
```

Comment:
If we still have case3.m, this file does not need to include these data format extensions.